tests/errors/binding/main-concrete.expect

```diff
@@ -0,0 +1,11 @@
+---CODE---
+1
+---STDERR---
+error: Default values for parameters in the main component must be concrete
+  ┌─ tests/errors/binding/main-default.fil:2:19
+  │
+2 │ comp main[?A=31, ?B=1+W]<'G:1>() -> () {
```

Comment:
The parameter `W` is not bound in this interface?

Reply:
Looks like this wasn't caught because it was superseded by the error about concreteness. Fixed!